### DISCUSSION TITLE
UIZepetoPlayerControl 의 move 값을 알고 싶습니다.

### DISCUSSION THREAD

**Discussion by peterkangdev** [naverz/zepeto-studio-kor#340] [5. Q&A]
UIZepetoPlayerControl 에서 touch event 의 값을 알고 싶습니다. 
이 값을 이용하여 움직임을 컨트롤 하고 싶은데, 값을 알 수가 없네요..

다른 방법도 있으면 알려주시면 감사하겠습니다.

**Comment by runasy-koonta** [comment]
Event를 Hook하여 어떻게든 얻어보려고 했는데 잘 안 되더라고요 ㅠㅠ
무식한 방법이지만 컨트롤러 노브의 x/y 값을 받는 방법이 제일 쉬운 것 같습니다.

**Reply by peterkangdev** [reply]
답변 감사합니다. 
근데. 어렵네요.
